recipes/astsearch/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "astsearch" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5db56de6ba7ecce1f5e5ba1fc6bf52a9bcf25ce330dc0decf001eb519e90de96
+
+build:
+  number: 0
```

Comment:
Any reason why this isn't `noarch: python`?

Reply:
@chrisburr It creates a console script. It was complaining on windows about not being able to be noarch.

I therefore removed the noarch.

Reply:
See https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points